packages/dataviews/src/dataviews-layouts/utils/item-click-wrapper.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import type { ReactNode, ReactElement, ComponentProps } from 'react';
+
+function getClickableItemProps< Item >( {
```

Comment:
Maybe we should call this `getClickableDefaultItemProps`. Or better, why not move it inline?